content/en/apps/tutorials/branding.md

```diff
@@ -55,9 +55,102 @@ The folder and files structure would look like this:
 
 Finally run the command: `cht --url=<intance-url> upload-branding`
 
-#### using the admin interface
+#### Using the admin interface
 
-Log in to your instance and navigate to `App Settings > Images > Icons`
+Log in to your instance and navigate to `Menu > App Settings > Images > Icons`
 
-{{< figure src="branding/menu.png" link="branding/menu.png" class="left col-6 col-lg-8" >}}
-{{< figure src="branding/images.png" link="branding/images.png" class="right col-6 col-lg-8" >}}
\ No newline at end of file
+![](menu.png)
+![](images.png)
+
+{{% alert title="Note" %}} 
+
+If your changes are not reflected on the browser, you will have to clear browser data to get rid of the already cached resources.
+
+Images used for the logo should be less than 100KB, transparent, have a high contrast, and horizonal in shape with ratio of about 3.5:1. We recommend SVG or PNG as an alternative image format.
+
+{{% /alert %}}
+
+### 2. Partner logos
+
+ If you would like to display a collection of logos representing all of the organizations and funders involved in a project, we have created a new space for these at the bottom of the About page which can be reached through `Menu > About`.
+
+ {{< figure src="about_page.png" link="about_page.png" class="center col-6 col-lg-8" >}}
+
+ #### Using cht-conf
+
+Create a `branding.json` file if it doesn't exist. (This may have already been created by the initialise-project-layout command).
+Edit the file with the following content:
+
+```json
+{
+   "resources": {
+     "partnerA": "parnerA.png",
+     "partnerB": "parnerB.png",
+     "partnerC": "parnerC.png"
+   }
+ }
+ ```
+
+ The folder and files structure would look like this:
+
+```
+./
+    partners.json
+    /partners
+        parnerA.png
+        parnerB.png
+```
+
+Finally run the command: `cht --url=<intance-url> upload-partners`
+
+#### Using the admin interface
+
+Log in to your instance and navigate to `Menu > App Settings > Images > Partners`
+
+![](menu.png)
+![](partners.png)
+
+### 3. Header tab icons
+
+As of cht-core 3.10, app header tabs icons are configurable. CHT currently has five tabs: messages, tasks, reports, contacts, analytics.
+
+{{< see-also page="apps/reference/app-settings/header_tabs" title="Header tabs" >}}
+
+
+#### Using cht-conf
+
+Create a `app_settings.json` file if it doesn't exist. (This may have already been created by the initialise-project-layout command).
```

Comment:
Create a `app_settings.json` file if it doesn't exist (this may have already been created by the `initialise-project-layout` command).

Might be good to mention `base_settings.json` from version 3.4.0+, when using the [modular approach](https://github.com/medic/cht-conf/blob/331c2d4a00c4c0361b6ea59f0029143c42675e62/release-notes.md#340).